hathor/cli/run_node.py

```diff
@@ -254,10 +255,31 @@ def create_wallet():
         for description in args.listen:
             self.manager.add_listen_address(description)
 
-        self.start_manager()
+        self.start_manager(args)
         self.register_resources(args)
 
-    def start_manager(self) -> None:
+    def start_sentry_if_possible(self, args: Namespace) -> None:
+        """Start Sentry integration if possible."""
+        if not args.sentry_dsn:
+            return
+        self.log.info('Starting Sentry', dsn=args.sentry_dsn)
+        try:
+            import sentry_sdk
+        except ModuleNotFoundError:
+            self.log.error('ERROR: You must install the package \'sentry-sdk\' to enable sentry.')
```

Comment:
I think this might be a bit confusing because `structlog-sentry` is also needed. But rather than making those dependencies explicit, I prefer to point to the `-E sentry` extras. And also I'm trying to avoid "ERROR" / "WARN" inside log messages.

I suggest something like:

```python
self.log.error('Please use `poetry install -E sentry` for enabling Sentry.')
```

Reply:
Nice suggestion. Done.